package.json

```diff
@@ -24,21 +24,27 @@
     "lint": "eslint .",
     "pretest": "npm run lint",
     "test": "mocha --async-only",
-    "cover": "nyc --reporter=lcov --reporter=text-summary npm test",
-    "coveralls": "npm run cover && istanbul-coveralls",
+    "cover": "nyc mocha --async-only",
     "respawn": "node test/bin/respawner --harmony test",
     "nospawn": "node test/bin/respawner test"
   },
-  "dependencies": {},
   "devDependencies": {
-    "eslint": "^2.13.0",
-    "eslint-config-gulp": "^3.0.1",
-    "expect": "^1.20.2",
-    "istanbul": "^0.4.3",
-    "istanbul-coveralls": "^1.0.3",
-    "mocha": "^3.5.3",
-    "nyc": "^11.3.0",
-    "v8flags": "^3.0.1"
+    "eslint": "^7.32.0",
+    "eslint-config-gulp": "^5.0.1",
+    "eslint-plugin-node": "^11.1.0",
+    "expect": "^27.3.1",
+    "mocha": "^8.4.0",
+    "nyc": "^15.1.0",
+    "v8flags": "^3.2.0"
```

Comment:
We'll need to release this and update to the next major

Reply:
I just released v8flags v4.0.0

Reply:
I got it.